docs/concepts/models/model_kinds.md

```diff
@@ -261,5 +261,273 @@ FROM db.employees;
+
+Target table will be updated with the following data:
+
+| ID | Name                | Price |     Updated At      |     Valid From      |      Valid To       |
+|----|---------------------|:-----:|:-------------------:|:-------------------:|:-------------------:|
+| 1  | Chicken Sandwich    | 10.99 | 2020-01-01 00:00:00 | 1970-01-01 00:00:00 | 2020-01-02 00:00:00 |
+| 1  | Chicken Sandwich    | 12.99 | 2020-01-02 00:00:00 | 2020-01-02 00:00:00 | 2020-01-03 00:00:00 |
+| 1  | Chicken Sandwich    | 14.99 | 2020-01-03 00:00:00 | 2020-01-03 00:00:00 |        NULL         |
+| 2  | Cheeseburger        | 8.99  | 2020-01-01 00:00:00 | 1970-01-01 00:00:00 | 2020-01-02 02:00:00 |
+| 2  | Cheeseburger        | 8.99  | 2020-01-03 00:00:00 | 2020-01-03 00:00:00 |        NULL         |
+| 3  | French Fries        | 4.99  | 2020-01-01 00:00:00 | 1970-01-01 00:00:00 |        NULL         |
+| 4  | Milkshake           | 3.99  | 2020-01-02 00:00:00 | 2020-01-02 00:00:00 | 2020-01-03 00:00:00 |
+| 4  | Chocolate Milkshake | 3.99  | 2020-01-03 00:00:00 | 2020-01-03 00:00:00 |        NULL         |
+
+Note: `Cheeseburger` was deleted from `2020-01-02 02:00:00` to `2020-01-03 00:00:00` meaning if you queried the table during that time range then you would not see `Cheeseburger` in the menu. This is the most accurate representation of the menu based on the source data provided. If `Cheeseburger` were added back to the menu with it's original updated at timestamp of `2020-01-01 00:00:00` then the `valid_from` timestamp of the new record would have been `2020-01-02 02:00:00` resulting in no period of time where the item was deleted. Since in this case the updated at timestamp did not change it is likely the item was removed in error and this again most accurately represents the menu based on the source data. 
+
+### Querying SCD Type 2 Models
+
+#### Querying the current version of a record
+
+Although SCD Type 2 models support history, it is still very easy to querying just the latest version of a record. Simply query the model as you would any other table. For example, if you wanted to query the latest version of the `menu_items` table you would simply run:
+
+```sql linenums="1" hl_lines="3"
+SELECT 
+    * 
+FROM 
+    menu_items
+WHERE 
+    valid_to IS NULL;
+```
+
+One could also create a view on top of the SCD Type 2 model that creates a new `is_current` column to make it easy for consumers to identify the current record.
+    
+```sql linenums="1" hl_lines="3"
+SELECT 
+    *, 
+    valid_to IS NULL AS is_current
+FROM
+    menu_items;
+```
+
+#### Querying for a specific version of a record at a give point in time
+
+If you wanted to query the `menu_items` table as it was on `2020-01-02 01:00:00` you would simply run:
+
+```sql linenums="1" hl_lines="3"
+SELECT 
+    *
+FROM
+    menu_items
+WHERE
+    id = 1
+    AND '2020-01-02 01:00:00' >= valid_from
+    AND '2020-01-02 01:00:00' < COALESCE(valid_to, '2199-12-31 23:59:59');
```

Comment:
AND ('...' < valid_to or valid_to is null), i don't like made up random numbers

Reply:
I see what you are saying but this example then goes with the view example of doing a COALESCE to support between. At the end of the day these are just examples and users can determine what makes sense in their given environment.

Reply:
i fundamentally disagree with your inclusion of them